spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java

```diff
@@ -175,7 +175,7 @@ private Map setupPySparkEnv() throws IOException{
     if (!env.containsKey("PYTHONPATH")) {
       SparkConf conf = getSparkConf();
       env.put("PYTHONPATH", conf.get("spark.submit.pyFiles").replaceAll(",", ":") +
-              ":../interpreter/lib/python");
+              ":../interpreter/lib/python:python");
```

Comment:
I have changed the location of Python libraries when launching python process to simplify the structure of libraries used in yarn process.